exercises/practice/binary-search-tree/.meta/example.py

```diff
@@ -5,7 +5,7 @@ def __init__(self, data, left=None, right=None):
         self.right = right
 
     def __str__(self):
-        fmt = 'TreeNode(data={}, left={}, right={})'
+        fmt = "TreeNode(data={}, left={}, right={})"
```

Comment:
```suggestion
        fmt = 'TreeNode(data={}, left={}, right={})'
```

Reply:
Our style is single quotes over double, except for docstrings, or where single doesn't work.